### DISCUSSION TITLE
What is console not defined

### DISCUSSION THREAD

**Discussion by pippo73** [revenz/Fenrus#198] [Q&A]
Hello all, 
I've just installed this app and I'm trying to figure out how it works.
I've add some apps, but when I try to test the API part, I get the message
"console not defined"
What does it mean?
thnx

**Comment by revenz** [comment]
what app are you testing when you get this error?  it will be a bug in that specific app.

**Comment by pippo73** [comment]
I've evolved since the beginning... actually I don't remember I did 1000 tests in this days with your platform... Sorry
